gui/src/app/Scripting/Runtime/makePyRuntime.ts

```diff
@@ -0,0 +1,65 @@
+import { ProjectDataModel } from "@SpCore/ProjectDataModel";
+
+import spPreamble from "./preamble.py?raw";
+import spRunData from "./run_data.py?raw";
+import spLoadConfig from "./load_args.py?raw";
+import spCmdStan from "./cmdstan.py?raw";
+import spRunSampling from "./sample.py?raw";
+import spDrawsScript from "../pyodide/sp_load_draws.py?raw";
+import spRunAnalysis from "./run_analysis.py?raw";
+
+const indent = (s: string) => {
+  return s
+    .trim()
+    .split("\n")
+    .map((x) => "    " + x)
+    .join("\n");
+};
+
+const makePyRuntimeScript = (project: ProjectDataModel) => {
```

Comment:
This kind of code is always going to be pretty ugly to write, but when you're ready to take this out of draft we can chat about whether there's anything to be done with it.